index.js

```diff
@@ -0,0 +1,82 @@
+'use strict';
+
+const DatastoreTrailpack = require('trailpack/datastore');
+const path               = require('path');
+const Wetland            = require('wetland').Wetland;
+
+module.exports = class WetlandTrailpack extends DatastoreTrailpack {
+
+  /**
+   * Validate configured stores
+   */
+  validate() {
+    if (!this.app.config.database || !this.app.config.database.stores) {
+      return this.app.log.logger.warn('No store configured.');
+    }
+
+    this.app.log.verbose('Trailpack-wetland is valid');
+  }
+
+  /**
+   * Instantiate wetland with config
+   */
+  configure() {
+    this.app.config.database.orm = 'wetland';
+
+    let stores       = this.app.config.database.stores;
+    let defaultStore = this.app.config.database.models.defaultStore;
+    let entities     = Object.keys(this.app.api.entities)
```

Comment:
Why like this? I'd expect an orm / wetland config for this. There's no way for me to configure the defaults for wetland now.